tutorials/llm/llama-3/pruning-distillation/01_data_preparation.ipynb

```diff
@@ -0,0 +1,102 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ab9e2e97-7f10-4353-859e-693842bde465",
+   "metadata": {},
+   "source": [
+    "### Step 1: Prepare the dataset\n",
+    "\n",
+    "The dataset has to be preprocessed using the [preprocess_data_for_megatron.py](https://github.com/NVIDIA/NeMo/blob/main/scripts/nlp_language_modeling/preprocess_data_for_megatron.py) script included in the NeMo Framework. This step will also tokenize data using the `meta-llama/Meta-Llama-3.1-8B` tokenizer model to convert the data into a memory map format.\n",
+    "\n",
+    "> `NOTE:` In the block of code below, pass the paths to your train, test and validation data files."
```

Comment:
fix punctuation

 In the block of code below, pass the paths to your train, test, and validation data files.